core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -16,8 +16,10 @@
 @ConfigRoot(phase = ConfigPhase.BUILD_TIME)
 public class NativeConfig {
 
-    public static final String DEFAULT_GRAALVM_BUILDER_IMAGE = "quay.io/quarkus/ubi-quarkus-native-image:22.1-java11";
-    public static final String DEFAULT_MANDREL_BUILDER_IMAGE = "quay.io/quarkus/ubi-quarkus-mandrel:22.1-java11";
+    public static final String GRAALVM_BUILDER_IMAGE_JAVA_11 = "quay.io/quarkus/ubi-quarkus-native-image:22.1-java11";
+    public static final String GRAALVM_BUILDER_IMAGE_JAVA_17 = "quay.io/quarkus/ubi-quarkus-native-image:22.1-java17";
+    public static final String MANDREL_BUILDER_IMAGE_JAVA_11 = "quay.io/quarkus/ubi-quarkus-mandrel:22.1-java11";
+    public static final String MANDREL_BUILDER_IMAGE_JAVA_17 = "quay.io/quarkus/ubi-quarkus-mandrel:22.1-java17";
```

Comment:
Small nitpick: I think we should have a constant for the version e.g. `22.1` so that we are sure it's consistent across the images. But we can do that next time we upgrade the images.